docs/docs/reference/inference.md

```diff
@@ -59,9 +59,9 @@
 
 ## Helpers
 
-::: sbi.inference.base.infer
+::: sbi.inference.trainers.base.infer
 
-::: sbi.inference.base.simulate_for_sbi
+::: sbi.inference.trainers.base.simulate_for_sbi
 
 ::: sbi.utils.user_input_checks.process_prior
```

Comment:
and add 
`::: sbi.utils.user_input_checks.process_simulator`

because it's required now for running things in parallel with `simulate_for_sbi`?